template/template.go

```diff
@@ -119,7 +118,7 @@ func Attr(selectorName string, ifAddrsRaw interface{}) (string, error) {
 func Parse(input string) (string, error) {
 	addrs, err := sockaddr.GetAllInterfaces()
 	if err != nil {
-		return "", errwrap.Wrapf("unable to query interface addresses: {{err}}", err)
+		return "", fmt.Errorf("unable to query interface addresses: {{err}}: %s", err)
```

Comment:
```suggestion
		return "", fmt.Errorf("unable to query interface addresses: %w", err)
```

(Same for all the others below)